app/workers/send_additional_reference_chase_email_to_candidates_worker.rb

```diff
@@ -0,0 +1,23 @@
+class SendAdditionalReferenceChaseEmailToCandidatesWorker
+  include Sidekiq::Worker
+
+  def perform
+    references_to_chase.each do |reference|
+      CandidateMailer.chase_reference_again(reference).deliver_later
+      ChaserSent.create!(chased: reference, chaser_type: :additional_reference_request)
```

Comment:
Two questions:

- I wonder if we need to check here if the application has been submitted. With the replacement feature it's possible for the candidate to have 2 complete references and 1 still waiting. I think that case we wouldn't want to send a nudge.
- This sends a nudge email per outstanding reference. I think we should perhaps send it per candidate? Otherwise the candidate would receive 2 identical emails if they have both references outstanding.

To avoid similar issues, can we print out a list of emails that would be sent on prod, so we can sense check it?

Reply:
Agreed on both counts. Will work something up. The existing nudge emails appear subject to the same issues, so will review those as well. 

Regarding this:

> To avoid similar issues, can we print out a list of emails that would be sent on prod, so we can sense check it?

Do you mean log these on each execution of the worker, or something else?

Reply:
Sorry, that sentence wasn't very clear :) 

I meant doing some kind of dry-run on production so that we can produce a list of emails that we will send once this is deployed, and have that reviewed by a PM.

As we don't backfill the existing applications, I'm expecting 50-60 emails to be sent after deploy (https://www.apply-for-teacher-training.education.gov.uk/integrations/performance-dashboard).

Reply:
@tijmenb New commits added with the suggested changes.